app/api/auth.py

```diff
@@ -224,15 +225,16 @@ def reset_password_post():
     try:
         user = User.query.filter_by(email=email).one()
     except NoResultFound:
-        return NotFoundError({'source': ''}, 'User not found').respond()
+        logging.info(email)
```

Comment:
Create a logger. Never use root logging module.

Also, logging the email makes no sense. What will I know from that. Properly log `Tried to reset password not existing email`